src/views/report/ScriptReportsOverview/index.tsx

```diff
@@ -367,27 +368,28 @@ const ScriptReportsOverview = withStyles(styles)(
             return !hasError ? (
                 <Box paddingBottom={5} marginX={2.8}>
                     <GenericList
-                        listActions={[
-                            {
-                                icon: <ReportIcon />,
-                                label: translator('script_reports.overview.list.actions.report'),
-                                onClick: (id) => {
-                                    const execution = listItems.find((listItem) => listItem.id === id);
-
-                                    redirectTo({
-                                        routeKey: ROUTE_KEYS.R_REPORT_DETAIL,
-                                        params: {
-                                            executionRequestId: id,
-                                            runId: execution && execution.data.runId,
-                                        },
-                                    });
-                                },
-                                hideAction: (id) => {
-                                    const execution = listItems.find((listItem) => listItem.id === id);
-                                    return execution.data.runId === null;
-                                },
-                            },
-                        ]}
+                        listActions={[].concat(
+                            checkAuthority(SECURITY_PRIVILEGES.S_SCRIPT_EXECUTIONS_READ, 'PUBLIC')
```

Comment:
Can we already use the security group name of the script in question instead of the fixed value 'PUBLIC'.
Also, privilege should be EXECUTION_REQUESTS_READ